zoom.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Zoom hardcodes ~/.config instead of using XDG_CONFIG_HOME.
+ZOOMUS=$HOME/.var/app/us.zoom.Zoom/.config/zoomus.conf
```

Comment:
I thought we'd have a shorthand for that. `XDG_CONFIG` or so. Maybe I am mistaken, though.